dango/testing/tests/lending.rs

```diff
@@ -41,21 +35,7 @@ fn feed_oracle_usdc_price(
     accounts: &mut TestAccounts,
     contracts: &Contracts,
 ) {
-    let id = PythId::from_str(USDC_USD_ID).unwrap();
     let precision = 6;
-    let usdc_denom = Denom::from_str("uusdc").unwrap();
-
-    // Register price source
-    suite
-        .execute(
-            &mut accounts.owner,
-            contracts.oracle,
-            &dango_types::oracle::ExecuteMsg::RegisterPriceSources(btree_map! {
-                usdc_denom.clone() => dango_types::oracle::PriceSource::Pyth { id, precision }
-            }),
-            Coins::default(),
-        )
-        .should_succeed();
```

Comment:
We now register all price sources during genesis n `setup_test()`, so no need to do it here.